libpod/storage.go

```diff
@@ -69,39 +69,25 @@ func (metadata *RuntimeContainerMetadata) SetMountLabel(mountLabel string) {
 // CreateContainerStorage creates the storage end of things.  We already have the container spec created
 // TO-DO We should be passing in an Image object in the future.
 func (r *storageService) CreateContainerStorage(ctx context.Context, systemContext *types.SystemContext, imageName, imageID, containerName, containerID string, options storage.ContainerOptions) (_ ContainerInfo, retErr error) {
-	var imageConfig *v1.Image
-	if imageName != "" {
```

Comment:
removing this breaks `--rootfs`, no? This should still check for `imageID != ""` which is the case if there is no image.

Reply:
Oops. Thanks, fixed.